src/components/Home/OtherSections/index.tsx

```diff
@@ -29,19 +29,25 @@ const OtherSections = () => {
           </div>
         </div>
       </section>
-      <section className={styles.vocabularySectionContainer}>
-        <div>
-          <HeadingDescription
-            headingContent="Increase your vocabulary"
-            descriptionContent="Traditional and new effective approaches to word study"
-          />
-          <Button outline={false} color="cyanLight" label="Textbook →" />
+      <section className="bg-tk_greyLight">
+        <div className="mx-auto grid sm:px-2 sm:py-4 md:px-8 md:py-10 lg:max-w-[1180px] lg:grid-cols-2 lg:px-0 lg:pb-0 lg:pt-[26px] xl:gap-[26px] xl:px-0 xl:pb-0">
+          <div className="mx-0 flex flex-col sm:ml-0 sm:py-4 md:py-[60px] lg:my-auto lg:ml-[100px] lg:block">
+            <HeadingDescription
+              headingContent="Increase your vocabulary"
+              descriptionContent="Traditional and new effective approaches to word study"
+            />
+            <Button outline={false} color="cyanLight" label="Textbook →" />
+          </div>
+          <img src="/vocabulary.svg" alt="Vocabulary" className="mx-0 my-auto sm:w-full" />
```

Comment:
should be `mx-auto my-auto sm:h-[200px] md:h-[363px] lg:h-auto`

Reply:
Hi Shawn,

The issues that you have mentioned have been fixed:
1. remove small media query for mobile as unnecessary
2. add missing height for mobile tablet and desktop by using h-[] or h-auto to control the image size
3. remove xl as there is no large desktop mode in figma design

Cheers